docs/DafnyRef/Introduction.md

```diff
@@ -60,7 +58,23 @@ initial section that deals with the lowest level constructs.
 
 The details of using (and contributing to) the dafny tool are described in the User Guide ([Section 13](#sec-user-guide)).
 
-## 1.1. Dafny Example {#sec-example}
+## 1.1. Dafny 4.0
+
+The most recent major version of the Dafny language is Dafny 4.0, released in February 2023.
+It has some backwards incompatibilities with Dafny 3, as decribed in the [migration guide](https://github.com/dafny-lang/ide-vscode/wiki/Quick-migration-guide-from-Dafny-3.X-to-Dafny-4.0).
+
+Some of the highlights of Dafny 4.0 are
+* support for full unicode
+TODO
+
+The user documentation has been expanded with more examples, A FAQ, and an error explanation catalog.
```

Comment:
```suggestion
The user documentation has been expanded with more examples, a FAQ, and an error explanation catalog.
```

Reply:
I would say 'a FAQ' because I pronounce it as an acronym, though I understand the pronunciation varies. Even then 'an FAQ' is worse than 'a F-A-Q'. Ii should not be uppercase though